### ISSUE TITLE
Investigate Next anti-cancer treatment date for ADSL

### ISSUE THREAD

**Issue by rossfarrugia** [pharmaverse/admiralonco#17]
### Feature Idea

Check possible specs for this later and see how this could be added to ADSL - likely derive_vars_merged_dt from admiral core would be sufficient, as likely CRF data collection for this would vary across companies

### Relevant Input

N/A

### Relevant Output

N/A

### Reproducible Example/Pseudo Code

N/A

**Comment by amitjaingsk** [comment]
Link to the workflow doc:
 https://exbpbox.box.com/s/crj05j1zx2cyh87khple99bywjg3bcjb

**Comment by bundfussr** [comment]
If the date should be merged to `ADSL`, `derive_vars_merged()` (single source) or `derive_var_extreme_dt()` (multiple sources) can be used.
If the date should be added as a new parameter, e.g., in `ADEVENT`, `derive_param_extreme_event()` can be used if there is just a single source. For the scenario of multiple sources we need to implement a new function.

**Comment by rossfarrugia** [comment]
Tangential discussion, but i spoke with Hinal earlier for an update on admiralonco and i wanted to add to this piece: if you end up making a vignette on NACT then it could make sense to pull the ADRS and ADTTE specific examples out of those vignettes and in to this one so its all in one place.

See https://pharmaverse.github.io/admiralonco/main/articles/adrs.html#anl01fl and https://pharmaverse.github.io/admiralonco/main/articles/adtte.html#creating-your-own-time-to-event-source-objects

**Comment by bundfussr** [comment]
> Tangential discussion, but i spoke with Hinal earlier for an update on admiralonco and i wanted to add to this piece: if you end up making a vignette on NACT then it could make sense to pull the ADRS and ADTTE specific examples out of those vignettes and in to this one so its all in one place.
> 
> See https://pharmaverse.github.io/admiralonco/main/articles/adrs.html#anl01fl and https://pharmaverse.github.io/admiralonco/main/articles/adtte.html#creating-your-own-time-to-event-source-objects

I agree assuming the examples in the ADRS and ADTTE vignette are replaced by links to the examples in the NACT vignette.

**Comment by bundfussr** [comment]
I have created an issue (https://github.com/pharmaverse/admiral/issues/1822) in admiral core for implementing the new function required for the case that there are multiple sources and that the date of anti-cancer therapy should be added to a BDS dataset like `ADEVENT`.
